Dockerfile

```diff
@@ -17,10 +17,12 @@ RUN \
     curl -sL https://deb.nodesource.com/setup_10.x | bash -; \
     apt-get install -y nodejs; \
     cd kinto/plugins/admin; npm install; npm run build; \
+#    pip3 install --upgrade pip \
+#    apt-get install -y httpie; \
```

Comment:
Maybe move this `httpie` install on the same line as `nodejs`

Reply:
It can be shifted to the upper line, however to the sake of readability it's just separated.
If you find it useful in the same manner, yeah, suit yourself :)